plugins/genshin/daily/material.py

```diff
@@ -57,27 +57,20 @@ def sort_item(items: List["ItemData"]) -> Iterable["ItemData"]:
 
     排序规则：持有（星级 > 等级 > 命座/精炼) > 未持有（星级 > 等级 > 命座/精炼）
     """
-    return (
-        ArkoWrapper(items)
-        .sort(lambda x: x.level or -1, reverse=True)
-        .groupby(lambda x: x.level is None)  # 根据持有与未持有进行分组并排序
-        .map(
-            lambda x: (
-                ArkoWrapper(x[1])
-                .sort(lambda y: y.rarity, reverse=True)
-                .groupby(lambda y: y.rarity)  # 根据星级分组并排序
-                .map(
-                    lambda y: (
-                        ArkoWrapper(y[1])
-                        .sort(lambda z: z.refinement or z.constellation or -1, reverse=True)
-                        .groupby(lambda z: z.refinement or z.constellation or -1)  # 根据命座/精炼进行分组并排序
-                        .map(lambda i: ArkoWrapper(i[1]).sort(lambda j: j.id))
-                    )
-                )
-            )
+
+    def key(item: ItemData):
```

Comment:
```suggestion
    def key(item: "ItemData"):
```

Reply:
Why this?

Reply:
Using variable 'ItemData' before assignment

Reply:
> Why this?

https://github.com/PaiGramTeam/PaiGram/blob/3ac0bdfec672f1691acedb884acd493532d936bf/plugins/genshin/daily/material.py#L55

Reply:
Are you looking for 'deferred type hint？